src/converters/lintConfigs/rules/ruleConverters/underscore-consistent-invocation.ts

```diff
@@ -0,0 +1,16 @@
+import { RuleConverter } from "../ruleConverter";
+
+export const convertUnderscoreConsistentInvocation: RuleConverter = (tslintRule) => {
+    return {
+        plugins: ["eslint-plugin-lodash"],
+        rules: [
+            {
+                ruleArguments: [
+                    tslintRule.ruleArguments.length === 0 || tslintRule.ruleArguments[0] === 'instance' ? 'never' : 'always',
+                    0,
```

Comment:
Based on the [Lodash plugin](https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/chaining.md#always), the "always" option is the one accepting a depth and should default to 3.

Reply:
Good catch. I think on our end we want to the converter to specify 0 for depth, since the TSLint rule will complain regardless of depth?